src/main/antlr/bio/terra/grammar/SQL.g4

```diff
@@ -64,8 +64,12 @@ expr : number
     | '(' expr ')'
     | column_expr
     | keyword
+    | cast_clause
     ;
 
+cast_clause
```

Comment:
Should we consider the azure equivalent here?

Reply:
It already works for azure!

String comparison Example:
`JSON_VALUE(variant_id, '$.val1') = N'1'`
Numeric comparison:
`JSON_VALUE(variant_id, '$.val1') > 1`

https://github.com/DataBiosphere/jade-data-repo/assets/13254229/66b6ab65-94ab-4f1a-8ff3-4571bd473f81

Reply:
oh sweet!